roles/ceph-osd/tasks/scenarios/lvm.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: use ceph-volume to create filestore osds with dedicated journals
```

Comment:
are all tasks in ceph-ansible excuted by root? all ceph-volume calls will need to be